### ISSUE TITLE
Add option to specify extra Packages

### ISSUE THREAD

**Issue by Sapd** [JacobDomagala/StaticAnalysis#4]
Hello,

this action is exactly what I searched for.

However I see no way to add dependency packages.
(E.g. my project needs the libhidapi-dev Ubuntu package).

Any way to easily add this?

Also you probably should publish the source of your main Dockerfile (https://hub.docker.com/r/jdomagala/static_analysis).

**Comment by JacobDomagala** [comment]
Hey, I've just merged the https://github.com/JacobDomagala/StaticAnalysis/pull/5 

Let me know if it works for you (it also includes the dockerfile that I'm using)

**Comment by potuz** [comment]
> Hey, I've just merged the #5
> 
> Let me know if it works for you (it also includes the dockerfile that I'm using)

Doesn't work for me, I need two packages (`yaml-cpp` and `libsnappy-dev`) either of them is installed if I request only one, but when both of them are requested (either space separated as in the README.md file or comma separated as in the PR description) the script bails out: 

```
Reading package lists...
+ apt-get install -y yaml-cpp,libsnappy-dev
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package yaml-cpp,libsnappy-dev
```

**Comment by JacobDomagala** [comment]
Thanks for pointing that out @potuz 
This should work now (using space separated, as described in Readme)
